code/game/objects/structures/flora.dm

```diff
@@ -818,6 +818,26 @@ GLOBAL_LIST_EMPTY(flora_uprooting_tools_typepaths)
 	. = ..()
 	icon_state = "bush[rand(1, 3)]"
 
+/obj/structure/flora/lunar_plant
+	name = "lunar plant"
+	desc= "This seemingly dead plant is actually quite alive, hibernating until sensing living things."
+	icon_state = "cave_plant"
+	icon = 'icons/obj/flora/xenoflora.dmi'
+	density = FALSE
+	light_color = COLOR_WHITE
+	light_range = 2
+
+/obj/structure/flora/lunar_plant/Initialize(mapload)
+	. = ..()
+	icon_state = "cave_plant[rand(1,3)]"
+
+/obj/structure/flora/lunar_plant/style_1
+	icon_state = "lunar_plant1"
+/obj/structure/flora/lunar_plant/style_2
+	icon_state = "lunar_plant2"
+/obj/structure/flora/lunar_plant/style_3
+	icon_state = "lunar_plant3"
```

Comment:
```suggestion
/obj/structure/flora/lunar_plant/style_1
	icon_state = "lunar_plant1"

/obj/structure/flora/lunar_plant/style_2
	icon_state = "lunar_plant2"

/obj/structure/flora/lunar_plant/style_3
	icon_state = "lunar_plant3"
```

Reply:
Everything in this file lacks newlines it would seem. I'll take this as reason to fix that.